lib/dom_utils.coffee

```diff
@@ -80,9 +84,7 @@ DomUtils =
 
         # eliminate invisible elements (see test_harnesses/visibility_test.html)
         computedStyle = window.getComputedStyle(element, null)
-        if (computedStyle.getPropertyValue('visibility') != 'visible' ||
-            computedStyle.getPropertyValue('display') == 'none')
-          continue
+        continue if computedStyle.getPropertyValue('visibility') != 'visible'
```

Comment:
Re. your earlier explanation: ok.
On this line, though, why are we dropping the test on `display`/`none`?